Rafael Beltran Hernandez

# Pregunta

a) Debido a que el elemento (2) y el elemento (6) son de ajuste H y h respectivamente, se tiene que el ajuste único es de 68 H7/h6 

In [2]:
# Tipo de Ajuste, se restan las tolerancias para un eje de 68 mm
dif= 30 - 19
print(dif)
# Juegos
juego_max =30 - 0
juego_min = 0
print('Los juegos máximos y mínimos son %a y %a' %(juego_max,juego_min))

11
Los juegos máximos y mínimos son 30 y 0


Como la diferencia de tolerancias es positiva, es un ajuste sin interferencia 

b) De acuerdo con la tabla del material de apoyo, se tiene que usar un ajuste 40H7/n6

# Pregunta

In [3]:
# Declaración de variables
D = 40 # Diámetro Base
desv_hueco = (25,0)
desv_eje = (-9,-25)

Juegos Máximos y mínimos del ajuste

In [4]:

juego_max = 25 - (-25)
juego_min = -9
print('Los juegos máximos y mínimos son %a y %a µm' %(juego_max,juego_min))

Los juegos máximos y mínimos son 50 y -9 µm


Diametros máximos y mínimos del eje y hueco

In [5]:
diametros_hueco=(D+desv_hueco[0],D+desv_hueco[1])
diametros_eje = (D + desv_eje[0], D + desv_eje[1])

print('Los diametros del hueco en mm son' ,diametros_hueco)
print('Los diametros del eje en mm son' ,diametros_eje)

Los diametros del hueco en mm son (65, 40)
Los diametros del eje en mm son (31, 15)


# Pregunta

In [6]:
# Declaracion de variables
s_carga = 10 # En %
s_diametro = 5
s_falla = 15

# el coeficiente del esfuerzo es
 
c_sigma = ( (1.1) / ( 0.95**2) )

# Coeficiente factor de seguridad 
c_n = 0.85

#factor de segudidad mínimo

n_min = c_sigma / c_n


** Procedimiento Matematico **

El esfuerzo normal máximo está dado por:
\begin{equation}
    \sigma_{max} = \frac{P_{max}}{A_min}  = 1.22 \sigma
\end{equation}

El factor de seguridad mínimo se da como
\begin{equation}
    n_{min} = \frac{S_min}{\sigma_max} = \frac{1.22}{0.85} = 1.44
\end{equation}

# Pregunta

A continuación se almacenan los datos

In [7]:
import numpy as np
S = np.array([93,95,97,99,101,103,105,107,109,111])
f = np.array([19,25,38,17,12,10,5,4,4,2])
prom = np.average(S, weights= f) # Calculo del Promedio ponderado
stdev= np.sqrt( np.average((S - prom)**2, weights= f) )
print('La los valores de promedio y desviación estándar son:',prom,stdev)

La los valores de promedio y desviación estándar son: 98.26470588235294 4.282720572455095


Cálculos de distribución normal:


In [8]:
import scipy.stats as stats
respuesta = stats.norm.ppf(0.01,prom,stdev)
print(respuesta)

88.30160798351106


De acuerdo con el resultado obtenido por la inversa de la distribución normal, se tiene que el valor correspondiente es de 88.3

# Pregunta

In [16]:
# Desviación factor de seguridad
s_T = 145 / 1500 # Desviación del torque porcentual
s_s = 23.5 / 312 # Desviación porcentual del esfuerzo de fluencia
s_n = np.sqrt(s_T**2 + s_s**2) # desviación del factor de seguridad
# z es el valor de la distribucion normal estándar 
z = stats.norm.ppf(0.99)
print(z)
# Carga y esfuerzo promedios
T = 1500
S = 312e+6


2.3263478740408408


Teniento en cuenta que estamos tomando una distribución normal estándar igual a una probabilidad $P_N = 0.99$ y tomando el $z$ de la inversa de esta distribución, $z=2.32$. Entonces la expresión del valor medio del factor de seguridad $\bar{n}$ es:

\begin{equation}
    \bar{n} = \frac{\bar{S_y}}{\tau_{max}} \quad \therefore \quad \bar{n} = \frac{\pi  \bar{S_y}  d^3}{16\bar{T}}
\end{equation}

Como encontramos el valor de $z$ tomando una estándar, el promedio tiende a cero. El valor del diámetro en función de la incertidumbre o desviación estándar, $z$, $\bar{T}$ y $\bar{S_y}$

\begin{equation}
    z \sigma = \frac{\pi  \bar{S_y} d^3}{16 \bar{T}} \quad \therefore \quad d = \sqrt[3]{ \frac{16 \bar{T} z \sigma }{\pi \bar{S_y}}   }
\end{equation}


In [17]:
# Cálculos

d = ( (16*T * z * s_n)/(np.pi * S) )**(1/3)

print('El diámetro es en metros',d)


El diámetro es en metros 0.019111466607380046


Una vez calculado el díametro y con el valor del factor de seguridad promedio, para que 